### PR TITLE
Dependency Updates

### PR DESCRIPTION
**Pull request by amywalkerdev**
**Overall change:**
Updates a bunch of dependencies:

1. @babel/plugin-transform-runtime   7.10.1
2. @babel/core   7.10.2
3. @babel/preset-react   7.10.1
4. @babel/plugin-proposal-object-rest-spread   7.10.1
5. @babel/preset-env   7.10.2
6. @babel/plugin-proposal-throw-expressions   7.10.1
7. @babel/runtime   7.10.2
8. @babel/polyfill   7.10.1
9. @testing-library/dom   7.7.3
10. @testing-library/jest-dom   5.9.0
11. enzyme-to-json   3.5.0
12. eslint-plugin-cypress   2.11.1
13. eslint-plugin-react-hooks   4.0.4
14. lint-staged   10.2.7

**Code changes:**

- Updates 14 dev dependencies in package.json and package-lock.json

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by DenisHdz — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
     "@bbc/psammead-locales": "^4.1.8",
     "@bbc/psammead-media-indicator": "^4.0.7",
     "@bbc/psammead-media-player": "^2.7.10",
-    "@bbc/psammead-most-read": "^4.1.2",
+    "@bbc/psammead-most-read": "^4.1.3",
```
Comment (thekp) [inline package.json]: ooo, I'm actually in the progress of confirming the changes from this bump with UX 👀 I've got a separate PR covering this: https://github.com/bbc/simorgh/pull/6711

Comment (amywalkerdev) [reply package.json]: You know I did mean to check if you were doing that, then forgot 😄 
I'll remove this one now

Comment (amywalkerdev) [reply package.json]: Removed and snapshot updates removed

**Review by thekp — APPROVED** [review]
Thanks Amy 🎉

**Pull request merged**